inception-layer-docmetadata/src/main/resources/META-INF/user-guide/annotation_create-annotations_docmeta.adoc

```diff
@@ -0,0 +1,6 @@
+= Document metadata
+
+To manage document-level annotations, access the **Document metadata** sidebar on the annotation
```

Comment:
I would like some more documentation before we remove the switch, especially some nice screenshots